docs/contribute/generate.md

```diff
@@ -26,11 +26,13 @@ is an example for DynamoDB.
 1. Clone the async-aws/aws repository.
 1. Run `composer install`
 1. Copy `./src/Service/.template` to `./src/Service/DynamoDb`
-1. Rename all occurrences of `Foobar` to `DynamoDb` and `foobar` to `dynamo-db`
+1. Replace all occurrences of `Foobar` to `DynamoDb` and `foobar` to `dynamo-db`
+1. Rename `FoobarClient` into `DynamoDbClient`
 1. Register the service in the `services` section of the `./manifest.json` file.
    1. Look at the official SDK for resources in `./src/data`. ([DynamoDB example](https://github.com/aws/aws-sdk-php/tree/3.133.23/src/data/dynamodb/2012-08-10))
    1. Add a link to `source`, and, if the files exists, links to `documentation`, `example`, `pagination`, `waiter` etc.
    1. Leave the `methods` key empty. (`"methods": []`)
+1. Add entry for the client in the `AsyncAws\Symfony\Bundle\DependencyInjection\AwsPackagesProvider::getAllServices` method.
```

Comment:
I decided not to add it here because there is a test that verifies that. There are a few more things todo, like update the ApiClientFactory and the readme. But I excluded those too.

Reply:
Indeed, I didn't ran tests before writing this line.. removed